modules/osd-create-cluster-red-hat-account.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * osd_quickstart/osd-quickstart.adoc
+// * osd_cluster_create/creating-an-aws-cluster.adoc
+// * osd_cluster_create/creating-a-gcp-cluster.adoc
+
+ifeval::["{context}" == "osd-getting-started"]
+:osd-quickstart:
+endif::[]
+ifeval::["{context}" == "osd-creating-a-cluster-on-aws"]
+:osd-on-aws:
+endif::[]
+ifeval::["{context}" == "osd-creating-a-cluster-on-gcp"]
+:osd-on-gcp:
+endif::[]
+
+:_content-type: PROCEDURE
+ifdef::osd-quickstart[]
+[id="create-cluster-red-hat-account_{context}"]
+= Creating a cluster with a Red Hat cloud account
+endif::osd-quickstart[]
+ifdef::osd-on-aws[]
+[id="osd-create-aws-cluster-red-hat-account_{context}"]
+= Creating a cluster on AWS with a Red Hat cloud account
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+[id="osd-create-aws-cluster-ccs_{context}"]
+= Creating a cluster on GCP with a Red Hat cloud account
+endif::osd-on-gcp[]
+
+Through {OCM}, you can create an {product-title} cluster 
+ifdef::osd-on-aws[]
+on {AWS} 
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+on {GCP} 
+endif::osd-on-gcp[]
+using a standard cloud provider account owned by Red Hat.
+
+.Procedure
+
+. Log in to {console-redhat-com} and click *Create Cluster*.
+
+. In the *Cloud* tab, click *Create Cluster* in the *Red Hat OpenShift Dedicated* row.
+
+. In the *Billing model* dialog, configure the subscription type and infrastructure type:
+.. Select a subscription type.
```

Comment:
The next step is "Select the *Red Hat cloud account* infrastructure type to deploy".  *Red Hat cloud account* infrastructure type only support subscription type "Annual". So it's better to just ask user select Annual subscription type.
And subscription type radio button will be hide once user have no quota, I wonder if we should clarify it here.

Reply:
@xueli181114 thank you for this comment. In the stage environment, I see two available subscription options for the Red Hat cloud account infrastructure type:

- Free trial (upgradeable)
- Annual: Fixed capacity subscription from Red Hat

I guess the available options depend on a customer's subscriptions. If I state only to select the "Annual" option, this might not cater for all use cases, such as the trial option. I have subsequently added the following note under the "Select a subscription type step":

> [NOTE]
> ====
> The subscription types that are available to you depend on your {product-title} subscriptions and resource quotas.
> ====

Reply:
There is no plan to update this right?

Reply:
@xueli181114 I added the note instead. Based on the details outlined in my comment, if I change the step to only reference annual billing, then it might not cater for every possible use case. As I mentioned, I see **Free trial** in the UI at this point too. Is there something that I am missing?

Reply:
@xueli181114 following on from our discussion on Slack, I understand now that the trial option becomes unavailable when you opt to use a Red Hat cloud account. I have subsequently updated the step in this procedure as follows:

```
. Under *Billing model*, configure the subscription type and infrastructure type:
.. Select the *Annual* subscription type. Only the *Annual* subscription type is available when you deploy a cluster using a Red Hat cloud account.
+
[NOTE]
====
You must have the required resource quota for the *Annual* subscription type to be available. For more information, contact your sales representative or Red Hat support.
====
```